src/views/favorites_panel.dot

```diff
@@ -13,20 +12,19 @@
       <h4 class="favorite_panel__container__title">
         {{= _n('%d favorite place', '%d favorite places', this.favoritePois.length, 'favorite panel') }}
       </h4>
-
       {{= this.filterPanel.render() }}
-
       <ul>
         <li class="favorite_panel__item">
 
           <div class="favorite_panel__swipe_window">
             <div class="favorite_panel__swipe_flow--{{= this.favoritePois.length }}">
               {{~ this.favoritePois:poi:i }}
-                <div class="favorite_panel__swipe_element" id="{{= `favorite_item_${i}` }}">
+                <div class="favorite_panel__swipe_element" {{= click(this.go, this, poi) }} id="{{= `favorite_item_${i}` }}">
```

Comment:
`cursor: pointer;` would be nice here too

Reply:
fixed